api.md

```diff
@@ -580,11 +580,13 @@ the same semantics as the `show_chunks` [function](#show_chunks).
 
 |Name|Description|
 |---|---|
-| `table_name` | Hypertable name from which to drop chunks. If not supplied, all hypertables are affected.
+| `table_name` | Hypertable name or continuous aggregate view from which to drop chunks.
 | `schema_name` | Schema name of the hypertable from which to drop chunks. Defaults to `public`.
 | `cascade` | Boolean on whether to `CASCADE` the drop on chunks, therefore removing dependent objects on chunks to be removed. Defaults to `FALSE`.
 | `cascade_to_materializations` | Set to `TRUE` to also remove chunk data from any associated continuous aggregates. Set to `FALSE` to only drop raw chunks (while keeping data in the continuous aggregates). Defaults to `NULL`, which errors if continuous aggregates exist.|
 
+>:TIP: The `table_name` argument was optional in previous versions, but this is now deprecated: `table_name` should always be given.
```

Comment:
Since hypertable name should be always given, all examples should use it. Otherwise, this statement is incorrect.

Reply:
Good point. Fixed.